### PR TITLE
Cmdlet Cleanup

### LINKED ISSUE #223: Wrongly named cmdlets
This issue it to keep track of cmdlets that are not properly named.

### PR DESCRIPTION
**Pull request by peombwa**
This PR closes #223 #75 #302 #318 by making the following changes:
- Applies custom directives to remove invalid paths from the OpenAPI docs.
- Applies directive that detects and removes duplicates in cmdlet names.
- Cleans up the module mapping config.

### CODE REVIEW

File: src/Identity.OnPremisesPublishingProfiles/Identity.OnPremisesPublishingProfiles/Microsoft.Graph.Identity.OnPremisesPublishingProfiles.psd1 [hunk 1]
```diff
@@ -71,39 +71,39 @@ FormatsToProcess =
 # NestedModules = @()
 
 # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
-FunctionsToExport = 'Get-MgOnPremisPublishingProfileAgent', 
+FunctionsToExport = 'Get-MgOnPremisPublishingProfile',
```
Comment (darrelmiller) [inline src/Identity.OnPremisesPublishingProfiles/Identity.OnPremisesPublishingProfiles/Microsoft.Graph.Identity.OnPremisesPublishingProfiles.psd1:74]: Should be OnPremise.  It's missing an e

File: src/Identity.OnPremisesPublishingProfiles/Identity.OnPremisesPublishingProfiles/readme.md [hunk 2]
```diff
@@ -32,6 +32,20 @@ title: $(service-name)
...
+
+### Directives
+
+> see https://github.com/Azure/autorest/blob/master/docs/powershell/directives.md
+
+``` yaml
+directive:
+# Rename cmdlets with duplicates in their name.
+  - where:
+      subject: ^(OnPremisPublishingProfile)(\1)+
```
Comment (darrelmiller) [inline src/Identity.OnPremisesPublishingProfiles/Identity.OnPremisesPublishingProfiles/readme.md:44]: Missing `e`

File: src/Identity.OnPremisesPublishingProfiles/Identity.OnPremisesPublishingProfiles/Microsoft.Graph.Identity.OnPremisesPublishingProfiles.psd1 (hunk 1 above)
Comment (peombwa) [reply src/Identity.OnPremisesPublishingProfiles/Identity.OnPremisesPublishingProfiles/Microsoft.Graph.Identity.OnPremisesPublishingProfiles.psd1:74]: This looks like a typo from AutoREST when getting a singular of `onPremises`. I'll open an issue on their end as well.

File: src/Identity.OnPremisesPublishingProfiles/Identity.OnPremisesPublishingProfiles/readme.md (hunk 2 above)
Comment (peombwa) [reply src/Identity.OnPremisesPublishingProfiles/Identity.OnPremisesPublishingProfiles/readme.md:44]: I'll fix the AutoREST typo via a directive.

**Pull request merged**